packages/tenant-process/src/services/tenantService.ts

```diff
@@ -556,6 +561,86 @@ export function tenantServiceBuilder(
       return updatedTenant;
     },
 
+    async revokeVerifiedAttribute({
+      tenantId,
+      attributeId,
+      organizationId,
+      correlationId,
+    }: {
+      tenantId: TenantId;
+      attributeId: AttributeId;
+      organizationId: TenantId;
+      correlationId: string;
+    }): Promise<Tenant> {
+      logger.info(
+        `Revoke verified attribute ${attributeId} to tenant ${tenantId}`
+      );
+
+      if (organizationId === tenantId) {
+        throw verifiedAttributeSelfRevocation();
+      }
+
+      await assertAttributeRevocationAllowed({
+        producerId: organizationId,
+        consumerId: tenantId,
+        attributeId,
+        readModelService,
+      });
+
+      const targetTenant = await retrieveTenant(tenantId, readModelService);
+
+      const verifiedTenantAttribute = targetTenant.data.attributes.find(
+        (attr) =>
+          attr.type === tenantAttributeType.VERIFIED && attr.id === attributeId
+      ) as VerifiedTenantAttribute;
+
+      if (!verifiedTenantAttribute) {
+        throw attributeNotFound(attributeId);
+      }
+
+      const verifier = verifiedTenantAttribute.verifiedBy.find(
+        (a) => a.id === organizationId
+      );
+
+      if (!verifier) {
+        throw attributeRevocationNotAllowed(tenantId, attributeId);
+      }
+
+      const revoker = verifiedTenantAttribute.revokedBy.find(
+        (a) => a.id === organizationId
+      );
+
+      if (revoker) {
+        throw attributeAlreadyRevoked(tenantId, organizationId, attributeId);
+      }
+
+      // eslint-disable-next-line functional/no-let
+      let updatedTenant: Tenant = {
+        ...targetTenant.data,
+        updatedAt: new Date(),
+      };
+
+      updatedTenant = updateAttribute(
+        {
+          updatedTenant,
+          targetTenant,
+          attributeId,
+          revocationTimestamp: new Date(),
+        },
+        verifiedTenantAttribute.assignmentTimestamp
+      );
+
+      const event = toCreateEventTenantVerifiedAttributeRevoked(
+        targetTenant.data.id,
+        targetTenant.metadata.version,
+        updatedTenant,
+        attributeId,
+        correlationId
+      );
+      await repository.createEvent(event);
```

Comment:
```suggestion
      await repository.createEvent(
        toCreateEventTenantVerifiedAttributeRevoked(
          targetTenant.data.id,
          targetTenant.metadata.version,
          updatedTenant,
          attributeId,
          correlationId
        )
      );
```

Reply:
We have implemented this convention every time we launch an event, if we decide to eliminate it we must refactor all the PR.
In case we open a different PR to do a general refactor, what do you think?

Reply:
> We have implemented this convention every time we launch an event, if we decide to eliminate it we must refactor all the PR. In case we open a different PR to do a general refactor, what do you think?

I know that sometimes we do like this and sometime we don't but I don't think it's enough to justify a refactor just for this, but while we are at it I think it's better to not declare a useless const